plugins/inputs/lustre2/lustre2.go

```diff
@@ -25,8 +25,8 @@ type tags struct {
 // Lustre proc files can change between versions, so we want to future-proof
 // by letting people choose what to look at.
 type Lustre2 struct {
-	Ost_procfiles []string `toml:"ost_jobstat"`
-	Mds_procfiles []string `toml:"mds_jobstat"`
+	Ost_procfiles []string
+	Mds_procfiles []string
```

Comment:
We should keep the struct tags, they just need corrected.  This is mostly style but I think it also blocks alternate capitalization like `Ost_procfiles = ["foo"]`